### PR TITLE
fix(sample): Update sample to avoid deprecated secretOrPrivateKey

### PR DESCRIPTION
**Pull request by saarw**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe: remove deprecation warning
```

## What is the current behavior?
Nest prints the following warning when signing a token
[JwtService] "secretOrPrivateKey" has been deprecated, please use the new explicit "secretOrKeyProvider" or use "privateKey"/"publicKey" exclusively.

Issue Number: N/A

## What is the new behavior?
No warning is printed

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: sample/19-auth/src/auth/auth.module.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { JwtStrategy } from './jwt.strategy';
   imports: [
     PassportModule.register({ defaultStrategy: 'jwt' }),
     JwtModule.register({
-      secretOrPrivateKey: 'secretKey',
+      secretOrKeyProvider: () => 'secretKey',
```
Comment (kamilmysliwiec) [inline sample/19-auth/src/auth/auth.module.ts]: We can use `secret: 'secretKey'` here :)

**Comment by kamilmysliwiec** [comment]
I left one comment. Could you please fix it? :)

**Comment by saarw** [comment]
Thanks! Done!

**Comment by kamilmysliwiec** [comment]
Thank you!

**Pull request merged**